crates/sui-json-rpc-types/src/sui_transaction.rs

```diff
@@ -178,13 +186,20 @@ impl SuiTransactionResponseOptions {
     }
 }
 
+#[serde_as]
 #[derive(Serialize, Deserialize, Debug, JsonSchema, Clone, Default)]
 #[serde(rename_all = "camelCase", rename = "TransactionResponse")]
 pub struct SuiTransactionResponse {
     pub digest: TransactionDigest,
     /// Transaction input data
     #[serde(skip_serializing_if = "Option::is_none")]
     pub transaction: Option<SuiTransaction>,
+    /// BCS encoded [SenderSignedData] that includes input object references
+    /// returns empty array if `show_raw_transaction` is false
+    #[serde_as(as = "Base64")]
+    #[schemars(with = "Base64")]
+    #[serde(skip_serializing_if = "Vec::is_empty", default)]
+    pub raw_transaction: Vec<u8>,
```

Comment:
I would love to make this `Option<Vec<u8>>` like the other field, but `#[serde_as(as = "Base64")]` does not work well with  `Option<Vec<u8>>`

Reply:
Awesome, I'll try this